code/modules/projectiles/projectile.dm

```diff
@@ -182,86 +215,105 @@
 	/// Slurring applied on projectile hit
 	var/slur = 0 SECONDS
 
-	var/dismemberment = 0 //The higher the number, the greater the bonus to dismembering. 0 will not dismember at all.
-	var/catastropic_dismemberment = FALSE //If TRUE, this projectile deals its damage to the chest if it dismembers a limb.
-	var/impact_effect_type //what type of impact effect to show when hitting something
-	var/log_override = FALSE //is this type spammed enough to not log? (KAs)
-	/// If true, the projectile won't cause any logging. Used for hallucinations and shit.
+	/// Damage the limb must have for it to be dismembered upon getting hit. 0 will prevent dismembering altogether
+	var/dismemberment = 0
+	/// If TRUE, this projectile deals its damage to the chest if it dismembers a limb.
+	var/catastropic_dismemberment = FALSE
+	/// Impact VFX created upon hitting something
+	var/impact_effect_type
+	/// If the act of firing this projectile does not create logs
+	var/log_override = FALSE
+	/// If true, the projectile won't cause any logging whatsoever. Used for hallucinations and shit.
 	var/do_not_log = FALSE
 	/// We ignore mobs with these factions.
 	var/list/ignored_factions
-
-	///If defined, on hit we create an item of this type then call hitby() on the hit target with this, mainly used for embedding items (bullets) in targets
+	/// Turf that we have registered connect_loc signal - this is done for performance, as we're moving ~a dozen turfs per tick
+	/// and registering and unregistering signal for every single one of them is stupid. Unregistering the signal from the correct turf in case we get moved by smth else is important
```

Comment:
```suggestion
	/**
	 * Turf that we have registered connect_loc signal - this is done for performance, as we're moving ~a dozen turfs per tick
	 * and registering and unregistering signal for every single one of them is stupid. Unregistering the signal from the correct turf in case we get moved by smth else is important
	 */
```